### ISSUE TITLE
Add isEmoji

### ISSUE THREAD

**Issue by MartinSStewart** [miniBill/elm-unicode#2]
I haven't understood what all the unicode categories mean but there doesn't seem to be a way to determine if a unicode character is an emoji. I had a use case for this once where I wanted to find the emoji usage frequency in some messages so I think it would be useful if there was an isEmoji function.

**Comment by miniBill** [comment]
This involves parsing https://unicode.org/Public/emoji/13.1/emoji-sequences.txt which is... doable but slightly annoying. Will do in the future. Possibly in a couple days

**Comment by MartinSStewart** [comment]
I agree, that's a strange format to parse. There's no rush though to add this. My use case was for something that I did a few months ago.

**Comment by miniBill** [comment]
From a private conversation:
> My particular use case is counting what emojis appear in a string so I can show commonly used ones at the top of an auto complete list. So I'd need to know the actual emoji, not the emoji components.
> 
> In other words, String -> List Emoji. Which probably not a very general purpose API unfortunately

> Yeah, not really
> You probably want String.foldl and a state machine based on https://unicode.org/Public/UCD/latest/emoji/emoji-sequences.txt

Unfortunately emojis are weird and therefore out of scope for now. If someone has a need for this we can have a conversation
